src/server/licence/postExitPage/index.njk

```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% extends 'layouts/page.njk' %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="gem-c-title__text govuk-heading-l"></h1>
+      <div class="govuk-body" data-testid="app-page-body">
+        <!-- put content here -->
```

Comment:
No content?

Reply:
this page is out of scope of this ticket, but i put the files needed for it in anyways ready for the next ticket